### PR TITLE
Add contribute.json file.

### PR DESCRIPTION
**Pull request by Osmose**
Add a contribute.json file a la http://contribute.paas.allizom.org/.

### CODE REVIEW

**Comment by Osmose** [comment]
@glogiotatidis r?

File: contribute.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "name": "about:home Snippets Service",
+    "description": "The Snippets Service hosts small chunks of HTML, CSS, and JavaScript that are displayed on about:home in Firefox, among other places.",
+    "repository": {
+        "url": "https://github.com/mozilla/snippets-service/",
+        "license": "BSD 3-Clause",
+        "tests": "https://travis-ci.org/mozilla/airmozilla/"
```
Comment (glogiotatidis) [inline contribute.json]: Oops

File: contribute.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+    "name": "about:home Snippets Service",
+    "description": "The Snippets Service hosts small chunks of HTML, CSS, and JavaScript that are displayed on about:home in Firefox, among other places.",
+    "repository": {
+        "url": "https://github.com/mozilla/snippets-service/",
+        "license": "BSD 3-Clause",
```
Comment (glogiotatidis) [inline contribute.json]: Interesting that this project is not MPL. Do you know more?

**Comment by glogiotatidis** [comment]
Yes, and... we need the lib you're [talking about here](https://github.com/mozilla/mozillians/pull/1177#issuecomment-77885681). I like to serve the file and that will enable the firefox addon too.

**Comment by glogiotatidis** [comment]
r+wc

File: contribute.json (hunk 2 above)
Comment (Osmose) [inline contribute.json]: Playdoh is BSD 3-Clause, thus most projects that were based on it are too. As to why Playdoh was BSD 3-Clause, I'm not really sure.

**Pull request closed without merging**